src/game.cpp

```diff
@@ -2247,7 +2247,7 @@ bool loadGame(const char *pGameToLoad, bool keepObjects, bool freeMem, bool User
 		strcat(aFileName, "game.js");
 		bool haveScript = PHYSFS_exists(aFileName);
 		if (haveScript) {
-			data = runMapScript(aFileName, gameRandU32(), true);
+			data = runMapScript(aFileName, gameRandU32(), false);
```

Comment:
Is this an intentional change? Not sure exactly what the preview parameter actually does.

Reply:
It is unrelated to this fix, but I think it makes sense, since we're not loading a preview in this function.
Looking at the code, when preview is set to false, afterMapLoad is called, which does some initialization. It doesn't seem to change anything though. I can remove it if you want.

Reply:
Seems like `runMapScript` ultimately passes it through to the map script.

If we're going to do this, makes sense to me to set this call to `false`.